lib/logstash/outputs/datadog_metrics.rb

```diff
@@ -73,7 +72,7 @@ def receive(event)
 
     dd_metrics = Hash.new
     dd_metrics['metric'] = event.sprintf(@metric_name)
-    dd_metrics['points'] = [[to_epoch(event.timestamp), event.sprintf(@metric_value).to_f]]
+    dd_metrics['points'] = [[event.timestamp.to_i, event.sprintf(@metric_value).to_f]]
```

Comment:
There was a reason to have this to_epoch method, described in https://github.com/elasticsearch/logstash/pull/690, is this no longer valid? if so I'd suggest also removing the to_epoch method, it's only used here